lib/_tokenUtils.ts

```diff
@@ -25,29 +25,27 @@ export const getCookie = (name: string): string => {
   return "";
 };
 
-export const ANONYMOUS_USER_TOKEN = "ANONYMOUS_USER_TOKEN";
-
-export function setUserToken(userToken: string | number): void {
-  if (userToken === ANONYMOUS_USER_TOKEN) {
-    if (!supportsCookies()) {
-      throw new Error(
-        "Tracking of anonymous users is only possible on environments which support cookies."
-      );
-    }
-    const foundToken = getCookie(COOKIE_KEY);
-    if (
-      !foundToken ||
-      foundToken === "" ||
-      foundToken.indexOf("anonymous-") !== 0
-    ) {
-      this._userToken = `anonymous-${createUUID()}`;
-      setCookie(COOKIE_KEY, this._userToken, this._cookieDuration);
-    } else {
-      this._userToken = foundToken;
-    }
+export function setAnonymousUserToken(): void {
+  if (!supportsCookies()) {
```

Comment:
if `_useCookie` is `false`, supportsCookies should return true, since no error is needed in that case